dex/src/jmh/java/io/crate/data/RowsBatchIteratorBenchmark.java

```diff
@@ -36,7 +36,7 @@
 import java.util.stream.StreamSupport;
 
 @BenchmarkMode(Mode.AverageTime)
-@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@OutputTimeUnit(TimeUnit.SECONDS)
```

Comment:
Because it took microseconds yielded results > 10_000 consistently. It should be milliseconds though